app/controllers/album.js

```diff
@@ -56,7 +64,19 @@ exports.buyAlbums = (req, res, next) => {
 
 exports.getAlbumsList = (req, res, next) => {
   logger.info(`getAlbumsList method start, request methods: ${req.method}, endpoint: ${req.path}`);
-  return findAllAlbumsBy(req.params)
+  const attributes = ['id', 'title'];
```

Comment:
its dont seem pretty to create a variable for this.
put it directyl in the next line without declaring it